blocks/init/src/Blocks/components/page-overlay/components/page-overlay-editor.js

```diff
@@ -22,11 +22,9 @@ export const PageOverlayEditor = (attributes) => {
 		selector(additionalClass, additionalClass),
 	]);
 
-	return (
-		<>
-			{pageOverlayUse &&
-				<div className={overlayClass}></div>
-			}
-		</>
-	);
+	if (!pageOverlayUse) {
+		return null;
+	}
+
+	return (<div className={overlayClass}></div>);
```

Comment:
```suggestion
	return (
		<div className={overlayClass}></div>
	);
```

```suggestion
	return <div className={overlayClass}></div>;
```

It so doesn't matter but just to have it